<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="DESKTOP-AKSHAY2" failures="1" tests="1" name="Default test(failed)" time="0.089" errors="0" timestamp="2022-07-27T12:02:09 IST">
  <testcase classname="testNGXMLStudy.FailedTestCases" name="c" time="0.015">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testNGXMLStudy.FailedTestCases.c(FailedTestCases.java:26)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- c -->
  <testcase classname="testNGXMLStudy.FailedTestCases" name="a" time="0.0">
    <ignored/>
  </testcase> <!-- a -->
  <testcase classname="testNGXMLStudy.FailedTestCases" name="b" time="0.0">
    <ignored/>
  </testcase> <!-- b -->
  <testcase classname="testNGXMLStudy.FailedTestCases" name="d" time="0.0">
    <ignored/>
  </testcase> <!-- d -->
  <testcase classname="testNGXMLStudy.FailedTestCases" name="e" time="0.0">
    <ignored/>
  </testcase> <!-- e -->
</testsuite> <!-- Default test(failed) -->
